tgui/packages/tgui/interfaces/GeneModder.js

```diff
@@ -0,0 +1,262 @@
+                margin: '0px',
+                'margin-left': '0px',
+              }}
+            />
+            <Button
+              content = {seed.name}
+              onClick = {() => act('eject_seed')}
+            />
+            <Button ml="3px"
+              icon="pen"
+              tooltip="Name Variant"
+              onClick={() => act('variant_name')}
+            />
+          </Flex.Item>
+        ) : (
+          <Flex.Item>
+            <Button
+              content = "None"
+              onClick = {() => act('eject_seed')}
+            />
+          </Flex.Item>
+        )
+        }
+        </LabeledList.Item>
+
+        <LabeledList.Item label="Data Disk">
+          {has_disk ? (
+            <Flex.Item>
+              <Button
+                content= {disk.name}
+                onClick={() => act('eject_disk')}
+              />
+            </Flex.Item>
+          ) : (
+            <Flex.Item>
+              <Button
+              content = "None"
+              onClick = {() => act('eject_disk')}
+            />
+            </Flex.Item>
+          )}
+        </LabeledList.Item>
+        </LabeledList>
+    </Section>
+  );
+};
+
+const CoreGenes = (props, context) => {
+  const { act, data } = useBackend(context);
+  const {
+    disk,
+    core_genes
+  } = data;
+
+  return (
+    <Collapsible
+      key= "Core Genes"
+      title= "Core Genes"
+      open>
+      {core_genes.map((gene) => (
+        <Flex
+          key={gene}
+          py="2px"
+          className="candystripe">
+          <Flex.Item width="100%" ml="2px">
+            {gene.name}
+          </Flex.Item>
+          <Flex.Item pr="5px">
+            <Button
+              content="Extract"
+              disabled={!disk?.can_extract}
+              icon="save"
+              onClick={() => act('extract', {id: gene.id})}
+            />
+          </Flex.Item>
+          <Flex.Item>
+            <Button
+              content="Replace"
+              disabled={!gene.is_type || !disk.can_insert}
+              icon="arrow-circle-down"
+              onClick={() => act('replace', {id: gene.id})}
+            />
+          </Flex.Item>
+        </Flex>
+      ))}
+    </Collapsible>
+  );
+};
+
+const ReagentGenes = (props, context) => {
+  const { data } = useBackend(context);
+  const { reagent_genes, has_reagent } = data;
+
+  return (
+    <OtherGenes
+      title = "Reagent Genes"
+      gene_set = {reagent_genes}
+      do_we_show = {has_reagent}
+    />
+  );
+};
+
+const TraitGenes = (props, context) => {
+  const { data } = useBackend(context);
+  const { trait_genes, has_trait } = data;
+
+  return (
+    <OtherGenes
+      title = "Trait Genes"
+      gene_set = {trait_genes}
+      do_we_show = {has_trait}
+    />
+  );
+};
+
+
+const OtherGenes = (props, context) => {
+  const {
+    title,
+    gene_set,
+    do_we_show
+  } = props;
+  const { act, data } = useBackend(context);
+  const {
+    disk
+  } = data;
```

Comment:
```suggestion
  const { disk } = data;
```
it's a single const assignment, so you really don't need to make it multiline